integration/bbi_expansion_test.go

```diff
@@ -17,14 +17,8 @@ func TestBBIExpandsWithoutPrefix(tt *testing.T) {
 	// INFO[0000] A total of 9 models have completed the initial preparation phase`
 	t := wrapt.WrapT(tt)
 
-	scenarios, err := InitializeScenarios([]string{
-		"bbi_expansion",
-	})
-	t.R.NoError(err)
-	t.R.Len(scenarios, 1)
-
-	scenario := scenarios[0]
-	scenario.Prepare(context.Background())
+	scenario := InitializeScenario(t, "bbi_expansion")
```

Comment:
This is an infinitely better solution than the awkward "loop and find" solution I had